Hearthstone Deck Tracker/Controls/UnavailableBattlegroundsTypes.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<UserControl x:Class="Hearthstone_Deck_Tracker.Controls.UnavailableBattlegroundsTypes"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:Hearthstone_Deck_Tracker.Controls" xmlns:hearthstone_deck_tracker="clr-namespace:Hearthstone_Deck_Tracker"
+             mc:Ignorable="d" 
+      d:DesignHeight="450" d:DesignWidth="800">
+
+    <Border BorderBrush="#141617" BorderThickness="1" Background="#23272a" Margin="0,5,0,0" Width="202" Visibility="{Binding UnavailableTypesVisibility, RelativeSource={RelativeSource AncestorType=local:UnavailableBattlegroundsTypes}}">
+        <StackPanel>
+            <Border Background="#1d3657" BorderBrush="#141617" BorderThickness="0,0,0,1">
+                <hearthstone_deck_tracker:HearthstoneTextBlock FontSize="14"
+                    Margin="7,2"
+                    Text="Missing Minion Types"
```

Comment:
"Missing Minion Types" is very verbose on the one hand and "Missing" also sounds unintentional or like a problem. Just "Unavailable" might be better? More concise and sounds more intentional.

Reply:
This will also need to be localized